### PR TITLE
Blog with Keyconf 24 invitation

### PR DESCRIPTION
**Pull request by mposolda**

### CODE REVIEW

**Review by francis-pouatcha — APPROVED** [review]

**Review by ahus1 — APPROVED** [review]
LGTM, thank you for this announcement. 

@mposolda - should I go ahead and merge it?

File: blog/2024/keyconf24-invitation.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+* *State of OIDC FAPI2:* Get the latest updates on OpenID Connect's Financial-grade API (FAPI) security profile.
+
+* *New Grant Type SPI & Token Exchange Endpoints:* Technical sessions on Keycloak's expanded capabilities and how to leverage them.
+
+* *Many more to be announced...*
+
+And of course, there will be ample opportunities for networking, knowledge sharing, and connecting with the vibrant Keycloak community and Keycloak maintainers.
+
+== Save the Date and Join Us!
```
Comment (tnorimat) [inline blog/2024/keyconf24-invitation.adoc:34]: It seems that L12-18 and L34-38 are the same sentences.

**Review by tnorimat — COMMENTED** [review]
@mposolda I added one comment. Could you check it?

File: blog/2024/keyconf24-invitation.adoc (hunk 1 above)
Comment (mposolda) [reply blog/2024/keyconf24-invitation.adoc]: Thanks Takashi! It was not exactly same, but little bit similar. The lines 12-18 were outdated, so removed them from the PR.

**Comment by mposolda** [comment]
@ahus1 @tnorimat @francis-pouatcha Thanks everyone for the review!

**Pull request merged**